src/main/java/org/javarosa/core/util/DataUtil.java

```diff
@@ -87,4 +87,11 @@ public static String[] splitOnColon(String s) {
     public static String[] splitOnPlus(String s) {
         return s.split("[+]");
     }
+
+    public static boolean isImageFile(String filename){
+        String lowerFilename = filename.toLowerCase();
+        return lowerFilename.endsWith(".jpg") || lowerFilename.endsWith(".jpeg") ||
+                lowerFilename.endsWith(".png") || lowerFilename.endsWith(".gif") ||
+                lowerFilename.endsWith(".bmp") || lowerFilename.endsWith(".webp");
+    }
```

Comment:
where is this list coming from ? . Also better to store this list in some sort of collection  for easy maintaineability.